aio/scripts/contributors/validate-data.js

```diff
@@ -15,7 +16,7 @@ const EXISTING_GROUPS = new Set(['Angular', 'GDE', 'Collaborators']);
 _main();
```

Comment:
Since `_main()` is now an async function, errors thrown inside `_main()` will just cause it to reject, leading to an unhandled rejection.

Before Node.js v15, unhandled rejections will only emit a warning and not cause the process to terminate with an error code (unless Node.js is run with the `--unhandled-rejections=strict` flag). See https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode.

You need to explicitly exit with an error on rejection:

```suggestion
_main().catch(err => {
  console.error(err);
  process.exit(1);
});
```